deploy/charts/cert-manager/templates/cainjector-vpa.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.cainjector.enabled }}
+{{- if .Capabilities.APIVersions.Has "autoscaling.k8s.io/v1" }}
```

Comment:
suggestion: We discussed this in standup today and we didn't think this was required - Capabilities.APIVersions messes with helm template, and it's fine for this to break loudly if someone requests the VPA and it's not available.